virttest/env_process.py

```diff
@@ -908,10 +911,16 @@ def preprocess(test, params, env):
         params["image_raw_device"] = "yes"
 
     if params.get("storage_type") == "lvm":
-        lvmdev = qemu_storage.LVMdev(params, base_dir, "lvm")
-        params["image_name"] = lvmdev.setup()
-        params["image_raw_device"] = "yes"
-        env.register_lvmdev("lvm_%s" % params["main_vm"], lvmdev)
+        for image in params.objects("images"):
```

Comment:
Since `images` is a per VM param, here it should iterate each VM as well in order to prepare every lvm volumes requested by this test.